pkg/ui/workspaces/cluster-ui/src/insights/workloadInsightDetails/transactionInsightDetails.tsx

```diff
@@ -75,7 +75,7 @@ export const TransactionInsightDetails: React.FC<
 
   useEffect(() => {
     const stmtsComplete =
-      stmts != null && stmts.length === txnDetails?.stmtExecutionIDs?.length;
+      stmts != null && stmts?.length === txnDetails?.stmtExecutionIDs?.length;
```

Comment:
nit: You don't need to add the `?` to the second part of the && here. If stmts is nullish the `stmts != null &&` will ensure the part after && doesn't get read.